Dockerfile.jobserver

```diff
@@ -0,0 +1,47 @@
+FROM golang:1.14.6-alpine AS builder
+
+RUN apk add --update --no-cache ca-certificates bash make gcc musl-dev git openssh wget curl
+
+WORKDIR /go/src/mattermod
+
+COPY go.mod .
+COPY go.sum .
+RUN go mod download
+
+COPY . .
+RUN CGO_ENABLED=0 make build
```

Comment:
See my comment below for renaming, if you change it there this also needs to change to `make build-jobserver`. and the same line in `Dockerfile` needs to change to `make build-mattermod`